swagger/swagger-invocation/invocation-core/src/main/java/org/apache/servicecomb/swagger/invocation/context/InvocationContext.java

```diff
@@ -68,6 +68,22 @@ public void addContext(Map<String, String> otherContext) {
     context.putAll(otherContext);
   }
 
+  public void mergeContext(InvocationContext otherContext) {
+    mergeContext(otherContext.getContext());
+  }
+
+  public void mergeContext(Map<String, String> otherContext) {
+    if (otherContext == null) {
+      return;
+    }
+    if (otherContext.size() > context.size()) {
+      otherContext.putAll(context);
```

Comment:
1.loop context and otherContext.putIfAbsent
2.still did not process highway transport?

Reply:
> 1.loop context and otherContext.putIfAbsent
> 2.still did not process highway transport?

new traceid created in invocation.onStart, then merge context. if only merge absent keys, the traceid is the new one.
can we put the merge context before invocation.onStart?